test/quantization/test_numeric_suite.py

```diff
@@ -22,3 +26,49 @@ def test_compare_weights(self):
         self.assertEqual(len(weight_dict), 1)
         for k, v in weight_dict.items():
             self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+        annotated_conv_bn_relu_model = AnnotatedConvBnReLUModel().eval()
+        annotated_conv_bn_relu_model.fuse_model()
+        quantized_annotated_conv_bn_relu_model = quantize(
+            annotated_conv_bn_relu_model, default_eval_fn, self.img_data
+        )
+        weight_dict = compare_weights(
+            annotated_conv_bn_relu_model.state_dict(),
+            quantized_annotated_conv_bn_relu_model.state_dict(),
+        )
+        self.assertEqual(len(weight_dict), 1)
+        for k, v in weight_dict.items():
+            self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+    def test_compare_model_stub(self):
+        r"""Compare the output of quantized conv layer and its float shadow module
+        """
+        # eager mode
+        annotated_conv_model = AnnotatedConvModel().eval()
+        quantized_annotated_conv_model = quantize(
+            annotated_conv_model, default_eval_fn, self.img_data
+        )
+        data = self.img_data[0][0]
+        module_swap_list = [torch.nn.Conv2d]
```

Comment:
Similar comments as above. If you make module_swap_list = [ torch.nn.Conv2d, nn.intrinsic.modules.fused.ConvReLU2d] , it should work.